azure-pipelines.yml

```diff
@@ -18,27 +18,30 @@ jobs:
     - bash: sudo chown -R $USER $CONDA
       displayName: Take ownership of conda installation
     - bash: |
+        set -ex
         conda create --name flake8_env --yes python=3.8
-        conda activate flake8_env
+        source activate flake8_env
```

Comment:
conda activate was actually not working in master with,
```
CommandNotFoundError: Your shell has not been properly configured to use 'conda activate'.
To initialize your shell, run

    $ conda init <SHELL_NAME>
```
and conda init was not working either. We were just not checking for exit status (now done with `set -ex`) So reverted to `source activate` as I don't want to debug this.

Reply:
one issue with `conda` is that it expects the shell function to be the entry point and not the executable. So if we have the executable in the PATH. This has already given me a headache in other places.